template-parts/content/content-single.php

```diff
@@ -56,16 +49,9 @@
 		<?php
 		the_content(
 			sprintf(
-				wp_kses(
-					/* translators: %s: Name of current post. Only visible to screen readers */
-					__( 'Continue reading<span class="screen-reader-text"> "%s"</span>', 'twentynineteen' ),
-					array(
-						'span' => array(
-							'class' => array(),
-						),
-					)
-				),
-				get_the_title()
+				/* translators: %s: Name of current post. Only visible to screen readers */
+				esc_html__( 'Continue reading %s', 'twentynineteen' ),
+				'<span class="screen-reader-text">"' . get_the_title() . '"</span>'
```

Comment:
This change needs to be reverted to support proper translations in all languages. See https://core.trac.wordpress.org/ticket/31979 and WordPress/twentysixteen#383 for background.